protoc-gen-connect-kotlin/src/main/kotlin/build/buf/protocgen/connect/Generator.kt

```diff
@@ -12,24 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package build.buf.protocgen.connect
+package com.connectrpc.protocgen.connect
```

Comment:
The files should be moved to the new package location too - `src/main/kotlin/com/connectrpc/protogen/connect/*`